python-manual/modules/ROOT/pages/performance.adoc

```diff
@@ -60,6 +60,160 @@ for i in range(1000):
 ----
 
 
+[[lazy-eager-loading]]
+== Don't fetch large result sets all at once
+
+When submitting queries that may result in a lot of result records, don't retrieve all records at once.
+The Neo4j server can retrieve records in batches, and the driver can receive one batch and _wait_ until it has been processed by the application before receiving another one from the server.
+Lazy-loading a result spreads out network traffic and memory usage.
+
+For convenience, xref:query-simple.adoc[`.execute_query()`] always retrieves all result records at once (it is what the `Eager` in `EagerResult` stands for).
```

Comment:
Well technically (off to a great start :upside_down_face:), you can get `.execute_query` to discard records instead of streaming them.

Reply:
I assume you mean with custom transformers?

Reply:
Yep, exactly.